db/patterns/wufoo.eno

```diff
@@ -0,0 +1,12 @@
+name: Wufoo
+category: customer_interaction
+website_url: https://www.wufoo.com/
+organization: surveymonkey
```

Comment:
Note: this currently breaks the tests, since "surveymonkey" is currently "usabilla": https://github.com/ghostery/trackerdb/blob/f9f42bb689173f8996da14d04a5cb36cef05994c/db/organizations/usabilla.eno

In this case, the traffic to this organization is small enough to justify to rename it to "surveymonkey" (a more intuitive name) without breaking history. I will do that and rebase the PR here. Then it should pass the tests.

Otherwise, the entry looks fine.

Reply:
Tests pass now after rebasing (rename came in https://github.com/ghostery/trackerdb/pull/583)